js/modules/aioseop_module.js

```diff
@@ -75,7 +75,12 @@ function countChars( field, cntfield ) {
 		cntfield.style.color = "#fff";
 		cntfield.style.backgroundColor = "#f00";
 	} else {
-		if ( cntfield.value > ( field_size - 91 ) ) {
+		if (field.name == 'aiosp_title') {
```

Comment:
@arnaudbroes from a coding standard point of view, the below would be preferable

1. yoda condition 
2. === instead of ==